src/Core/Models/Business/OrganizationUserInvite.cs

```diff
@@ -20,8 +20,12 @@ public OrganizationUserInvite(OrganizationUserInviteRequestModel requestModel)
             Emails = requestModel.Emails;
             Type = requestModel.Type.Value;
             AccessAll = requestModel.AccessAll;
-            Collections = requestModel.Collections.Select(c => c.ToSelectionReadOnly());
```

Comment:
Why not simplify to the following:
```c#
Collections = requestModel.Collections?.Select(c => c.ToSelectionReadOnly());
```

Reply:
![giphy](https://user-images.githubusercontent.com/15897251/104505259-9f975700-55b1-11eb-9f30-487d030a80d4.gif)